app/controllers/mixins/actions/vm_actions/rename.rb

```diff
@@ -29,118 +24,19 @@ def build_rename_screen
           @in_a_form = true
         end
 
-        def name_changed
-          return unless load_edit("vm_rename__#{params[:id]}")
-
-          rename_get_form_vars
-          @changed = changed = session[:changed] = @edit[:new] != @edit[:current]
-          render :update do |page|
-            page << javascript_prologue
-            page << javascript_for_miq_button_visibility(changed)
-            page << "miqSparkle(false);"
-          end
-        end
-
-        # Set form variables for rename
-        def rename_set_form_vars
-          @edit = {}
-          @edit[:vm_id] = @record.id
-          @edit[:new] = {}
-          @edit[:current] = {}
-          @edit[:explorer] = true if params[:action] == 'x_button' || session.fetch_path(:edit, :explorer)
-          @edit[:current][:name] = @edit[:new][:name] = @record.name
-          @edit[:key] = "vm_rename__#{@record.id}"
-          session[:edit] = @edit
-        end
-
-        # Get form variable for rename
-        def rename_get_form_vars
-          @record = VmOrTemplate.find_by(:id => @edit[:vm_id])
-          @edit[:new][:name] = params[:name] if params[:name]
-          session[:edit] = @edit
-        end
-
         # Button actions for VM's renaming screen
         def rename_vm
           return unless load_edit("vm_rename__#{params[:id]}")
-
           @explorer = true if @edit[:explorer]
-          rename_get_form_vars
-          case params[:button]
-          when 'cancel'
-            rename_cancel
-          when 'save'
-            rename_save
-          when 'reset'
-            rename_reset
-          else
-            build_rename_screen
-            if @edit[:explorer]
-              replace_right_cell
-            else
-              render :action => 'rename'
-            end
-          end
-        end
-
-        def rename_finished
-          task_id = session[:async][:params][:task_id]
-          vm_name = session[:edit][:current][:name]
-          task = MiqTask.find(task_id)
-          if MiqTask.status_ok?(task.status)
-            add_flash(_("Rename of Virtual Machine \"%{name}\" has been initiated") % { :name => vm_name })
-          else
-            add_flash(_("Unable to rename Virtual Machine \"%{name}\": %{details}") %
-                      { :name => vm_name, :details => task.message }, :error)
-          end
-          session[:edit] = nil
+          build_rename_screen
           if @edit[:explorer]
-            @sb[:action] = nil
             replace_right_cell
           else
-            flash_to_session
-            javascript_redirect(previous_breadcrumb_url)
+            render :action => 'rename'
```

Comment:
Is this new line? why is this added?

Reply:
I think I just missed it when I was deleting lines. I will remove and repush.